client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-how-to-migrate/index.tsx

```diff
@@ -76,29 +77,41 @@ const SiteMigrationHowToMigrate: FC< Props > = ( props ) => {
 		? true
 		: false;
 
-	const { updateMigrationStatus } = useUpdateMigrationStatus();
+	const [ isLoading, setIsLoading ] = useState( false );
```

Comment:
Instead of an extra state that we need to control, we can use the `isLoading` returned from the [mutation](https://tanstack.com/query/v4/docs/framework/react/reference/useMutation).